src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -129,6 +129,15 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, ClrDa
     return this._field;
   }
 
+  /*
```

Comment:
I don't see any actual changes in this file, can you revert it?

Reply:
This is the most important change in the pull-request moving the `clrDgColType` above `clrDgField`

Reply:
Also alignment of the comment block - but that is not that important